commit d621292fb2c8157d9899dcd83fd04dd250e30fe4	d621292fb2c8157d9899dcd83fd04dd250e30fe4
Author: Michael Vrhel <michael.vrhel@artifex.com>
Date:   Thu Dec 29 12:00:40 2016 -0800

    Bug 697444 Unmatched transparency group pop
    
    This issue can only occur if there is an unmatched group pop.
    If the interpreter is doing that, then the interpreter is
    broken.  With this bug the user is intentionally doing it.
    We now throw and error when it occurs.

diff --git a/base/gdevp14.c b/base/gdevp14.c
index bec5a07c0..fd56ec975 100644
--- a/base/gdevp14.c
+++ b/base/gdevp14.c
@@ -1066,6 +1066,9 @@ pdf14_pop_transparency_group(gs_gstate *pgs, pdf14_ctx *ctx,
     gx_color_index drawn_comps = pdev->drawn_comps;
     bool nonicc_conversion = true;
 
+    if (nos == NULL)
+        return_error(gs_error_unknownerror);  /* Unmatched group pop */
+
     nos_num_color_comp = nos->parent_color_info_procs->num_components - nos->num_spots;
     tos_num_color_comp = tos_num_color_comp - tos->num_spots;
 
